### PR TITLE
MGMT-8504: Fix InfraEnv image updated event loop

### PR DESCRIPTION
**Pull request by nmagnezi**
# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -1446,13 +1448,11 @@ func (b *bareMetalInventory) createAndUploadNewImage(ctx context.Context, log lo
 
 	// return without generating the image if we're using the image service or if the image has already been generated
 	if b.ImageServiceBaseURL != "" {
-		if err = b.updateExternalImageInfo(infraEnv, infraEnvProxyHash, imageType); err != nil {
+		logrus.Info("XXXX: before updateExternalImageInfo")
```
Comment (carbonin) [inline internal/bminventory/inventory.go:1451]: @nmagnezi think you left a debug line in here.

Comment (nmagnezi) [reply internal/bminventory/inventory.go:1451]: yikes. removing.

Comment (nmagnezi) [reply internal/bminventory/inventory.go]: done

**Review by carbonin — APPROVED** [review]

**Pull request merged**